content/applications/hr/payroll/work_entries.rst

```diff
@@ -0,0 +1,172 @@
+
+.. image:: work_entries/validate.png
+   :align: center
+   :alt: Edit and/or validate a duplicate time-off request.
+
+Once the duplicate time-off request has been approved and saved, the screen goes back to the
+conflict. Click :guilabel:`Refuse Time Off` or :guilabel:`Approve Time Off` via the buttons in the
+top right to either approve or deny the request. Repeat for all conflicts until there are no
+conflicts to resolve.
+
+After conflicts have been resolved, work entries must be regenerated for each employee by clicking
+the :guilabel:`Regenerate Work Entries` button, and entering the corresponding information for each
+employee.
+
+.. image:: work_entries/regenerate-employee.png
+   :align: center
+   :alt: Regenerate each employee who has had work entires modified.
+
+Generating payslips
+===================
+
+To generate payslips, navigate to the time period the payslips should be generated for, either day,
+week, or month. When the desired pay period is displayed, click the :guilabel:`Generate Payslips`
+button.
+
+   .. image:: work_entries/generate-payslips.png
+      :align: center
+      :alt: Create payslips by clicking the Generate Payslips button.
+
+.. tip::
+   If the :guilabel:`Generate Payslips` button is not active (appears light turquoise instead of
+   dark turquoise), that indicates there are conflicts. *Solve conflicts first* will appear as a
+   warning when :guilabel:`Generate Payslips` is moused over. Resolve all conflicts before
+   generating payslips.
+
+A batch entry will appear for the time period selected. The batch name appears at the top in the
+:guilabel:`Name` field, typically listing the month and year for the particular batch.
+
+The date range the payslips apply to appears in the :guilabel:`Period` field. The company appears in
+the :guilabel:`Company` field, along with an option to mark the payslips as a credit note. To make
+changes, click the :guilabel:`Edit` button in the top left, make any changes, then click either
+:guilabel:`Save` to accept the changes or :guilabel:`Discard` to revert to the original data.
+
+   .. image:: work_entries/batch.png
+      :align: center
+      :alt: Batch information that appears when making a batch.
+
+Click the :guilabel:`Create Draft Entry` button to create the payslips for the batch.
+
+Click on the :guilabel:`Payslips` button in the top right to view all the payslips for the batch.
+
+.. Note::
+   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button has been
+   clicked. After, the payslip status will change to *Done*.
+
+Payslips can be printed by clicking the box next to each payslip to print, or clicking the box next
+to :guilabel:`Reference` to select all the payslips at once. Click the :guilabel:`Print` button, and
+a PDF file will be created with all the specified payslips.
+
+   .. image:: work_entries/print-payslips.png
+      :align: center
+      :alt: Click the Print button to print the payslips.
+
```

Comment:
It may be a good idea to link to the work entry configuration on Payroll.rst, as well as link to this doc from that section as well, WDYT?
```suggestion
.. seealso::
   :ref:`Configure work entries <payroll/work-entries-config>`

```

Reply:
YES! Mor info, the better =)